.travis.yml

```diff
@@ -2,15 +2,16 @@ language: scala
 sudo: required
 jdk:
   - oraclejdk8
+addons:
+  apt:
+    packages:
+      - oracle-java8-set-default # Important to add for removing Unsupported major.minor version 52.0 error
```

Comment:
How about: "Installing Elasticsearch from .deb causes default JDK in the system to fallback. This is required to ensure that 'java' (and other JDK commands) is linked to those of oraclejdk8"